tensorflow_datasets/core/download/kaggle.py

```diff
@@ -46,10 +47,44 @@
 class KaggleFile(object):
   """Represents a Kaggle competition file."""
   _URL_PREFIX = "kaggle://"
+  kaggle_type = collections.namedtuple(
```

Comment:
New class should be declared outside the `KaggleFile` scope in the global space. Also as this is anew type, it should be CamelCase.